package.json

```diff
@@ -1,9 +1,9 @@
 {
-    "name": "vscode-cosmosdb",
+    "name": "vscode-azuredatabases",
```

Comment:
Don't change the name yet. That has lots of ramifications that we should discuss/handle separately